ieee-cdf/ieee-cdf-model/src/main/java/com/powsybl/ieeecdf/model/IeeeCdfBranch.java

```diff
@@ -245,6 +245,10 @@ public int getzBusNumber() {
         return zBusNumber;
     }
 
+    public int getBus3Number() {
```

Comment:
You don't need to add this method to IEEE and Matpower model (and we should not as 3 windings is not part of their spec).

Reply:
OK, proposed modification